packages/router/package.json

```diff
@@ -33,7 +33,7 @@
   "author": "Ionic Team",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/ionic-team/stencil-router"
+    "url": "https://github.com/stencil-community/stencil-router"
```

Comment:
should we possibly do `"https://github.com/stencil-community/stencil-router/issues"` here?

Reply:
🤦 Yup! Done in https://github.com/stencil-community/stencil-router/pull/138/commits/e6ffbc1cb4dc83872a7ba8a49d2ca2911eedd56a